mipsevm/run_unicorn.go

```diff
@@ -130,6 +132,10 @@ func GetHookedUnicorn(root string, ram map[uint32](uint32), callback func(int, u
 			rt := value
 			rs := addr64 & 3
 			addr := uint32(addr64 & 0xFFFFFFFC)
+			if outputfault && addr == 0x30000804 {
```

Comment:
Would be nice to be able to specify the memory location for the fault.

Reply:
But if writing to random memory locations, two possible snafus:
- it crashes the program
- it mutates the memory but the output hash (doesn't include MIPS memory) is the same
→ Could modify the runner to compute & output the MIPS memory Merkle root for comparison.